notebooks/day1/01-basics-extra.jl

```diff
@@ -5,7 +5,7 @@ using Markdown
 using InteractiveUtils
 
 # ╔═╡ 4109a39c-00b7-4954-bb73-35407f82a6f7
-using Pkg; Pkg.add(url="https://github.com/Beramos/DS-Julia2925")
+using Pkg; Pkg.activate("../..")
```

Comment:
why is the DSJulia package environment activated here?
I thought we were going to keep these separated?
i.e. the env in the DSJulia only for the things in DSJulia and the env in notebooks for everything else you need in the course

Reply:
resolved in  4f01f53